frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,297 @@
+                  name="emailAuthUsername"
+                  data-test-id="email-auth-username"
+                  value={formValues.emailAuthUsername}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailAuthUsername: e.target.value },
+                    })
+                  }
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Password (Using LOGIN and PLAIN)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-password-help"
+                  name="emailAuthPassword"
+                  data-test-id="email-auth-password"
+                  value={formValues.emailAuthPassword}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailAuthPassword: e.target.value },
+                    })
+                  }
+                />
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Identity (Using PLAIN)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-auth-identity-help"
+                  name="emailAuthIdentity"
+                  data-test-id="email-auth-identity"
+                  value={formValues.emailAuthIdentity}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailAuthIdentity: e.target.value },
+                    })
+                  }
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Secret (CRAM-MDS)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-secret-help"
+                  name="emailAuthSecret"
+                  data-test-id="email-auth-secret"
+                  value={formValues.emailAuthSecret}
+                  onChange={(e) =>
+                    dispatchFormChange({
+                      type: 'setFormValues',
+                      payload: { emailAuthSecret: e.target.value },
+                    })
+                  }
+                />
+              </div>
+            </div>
+          </div>
+          <label className="co-no-bold">
```

Comment:
Use console checkbox classes instead of `co-no-bold`

Reply:
fixed